funtofem/driver/fun3d_oneway_driver.py

```diff
@@ -537,17 +533,12 @@ def _solve_unsteady_adjoint(self, scenario, bodies):
 
         # initialize, run, and do post adjoint
         self.solvers.flow.initialize_adjoint(scenario, bodies)
-        for rstep in range(scenario.steps + 1):
+        # one extra step here to include step 0 calls (see fully coupled driver)
+        for rstep in range(1, scenario.steps + 2):
             step = scenario.steps + 1 - rstep
             self.solvers.flow.iterate_adjoint(scenario, bodies, step=step)
             self._extract_coordinate_derivatives(scenario, bodies, step=step)
         self.solvers.flow.post_adjoint(scenario, bodies)
-        self._extract_coordinate_derivatives(scenario, bodies, step=0)
```

Comment:
What happened to _extract_coordinate_derivatives?

Reply:
It was ran after the `post_adjoint` which will cause an error (this is in the unsteady case which has not been tested in the `fun3d_oneway_driver`). Also, I increased the # of steps by 1 so it is still running the `extract_coordinate_derivatives` for step 0.